src/code/providers/google-drive-provider.coffee

```diff
@@ -195,8 +211,10 @@ class GoogleDriveProvider extends ProviderInterface
   _loadGAPI: ->
     if not window._LoadingGAPI
       window._LoadingGAPI = true
-      window._GAPIOnLoad = ->
-        @window._LoadedGAPI = true
+      window._GAPIOnLoad = =>
+        window._LoadedGAPI = true
+        # preload clients to avoid user delay later
+        @_loadedGAPI ->
```

Comment:
You set @_loadedGAPI to false above but `@_loadedGAPI ->` will transpile to a function call like this `this._loadedGAPI(function() {});`